apps/notifications/src/panel/boot/stylesheets/shared/colors.scss

```diff
@@ -1,3 +1,5 @@
 $wpnc__yellow-dark: var(--color-warning-60);
 $wpnc__red-darker: var(--color-error-90);
 $wpnc__yellow-lighter: var(--color-warning-0);
+$wpnc__rich-blue: #3858e9;
```

Comment:
Do we have these defined as scss vars as well?

Reply:
I couldn't fine them anywhere!